test/e2e/exists_test.go

```diff
@@ -39,6 +39,7 @@ var _ = Describe("Podman image|container exists", func() {
 		Expect(session).Should(Exit(0))
 	})
 	It("podman image exists in local storage by short name", func() {
+		Skip("FIXME-8165: shortnames don't seem to work with quay")
```

Comment:
This seems... pretty large? Any idea what's going on?

Reply:
I don't, not on this or any of the other two FIXMEs. They worry me too, but my goal was to get tests passing and then dive deeper. I lost too much time on the `map` problem. I can dive deeper into these other problems today, I just want to know if the approach in this PR is reasonable.

Reply:
I have no issue with the approach - it seems like a quick and easy fix to get things migrated away from Dockerhub.

Reply:
@mheon see #8176. Another one that was unexpectedly difficult to reproduce.